### ISSUE TITLE
Encryption in transit  

### ISSUE THREAD

**Issue by ofekhub88** [awslabs/aws-fluent-plugin-kinesis#218]
With this plugin .
IS the data encrypted between on premise aggregator and   AWS kinesis ?

**Comment by simukappu** [comment]
Yes, it is encrypted in transit. All Amazon Kinesis API including PutRecords are provided as HTTPS. This fluentd plugin is using AWS SDK for Ruby and AWS SDK internally calls Amazon Kinesis API.
